src/main/java/com/atlassian/shele/shelePlugin/customfield/SecondCustomField.java

```diff
@@ -0,0 +1,45 @@
+package com.atlassian.shele.shelePlugin.customfield;
+
+import com.atlassian.jira.component.ComponentAccessor;
+import com.atlassian.jira.issue.Issue;
+import com.atlassian.jira.issue.customfields.impl.GenericTextCFType;
+import com.atlassian.jira.issue.customfields.manager.GenericConfigManager;
+import com.atlassian.jira.issue.customfields.persistence.CustomFieldValuePersister;
+import com.atlassian.jira.issue.fields.CustomField;
+import com.atlassian.jira.issue.fields.layout.field.FieldLayoutItem;
+import com.atlassian.jira.issue.history.ChangeItemBean;
+import com.atlassian.plugin.spring.scanner.annotation.imports.ComponentImport;
+
+import javax.annotation.Nonnull;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.atlassian.shele.shelePlugin.utilit.Utilities.CUST_FIELD;
+
+public class SecondCustomField extends GenericTextCFType {
+
+    public SecondCustomField(@ComponentImport CustomFieldValuePersister customFieldValuePersister,
+                             @ComponentImport GenericConfigManager genericConfigManager) {
+        super(customFieldValuePersister, genericConfigManager);
+    }
+
+    @Nonnull
+    @Override
+    public Map<String, Object> getVelocityParameters(final Issue issue, final CustomField field,
+                                                     final FieldLayoutItem fieldLayoutItem) {
+        Map<String, Object> params = new HashMap<>();
+        String reportKey = issue.getReporter().getKey();
+        params.put("reportKey", reportKey);
+        return params;
+    }
+
+    @Override
+    public String getValueFromIssue(final CustomField field, final Issue issue) {
+        List<ChangeItemBean> itemBeans = ComponentAccessor.getChangeHistoryManager()
+                .getChangeItemsForField(issue, CUST_FIELD);
```

Comment:
does it actually count the number of assignees? what if we perform unassign actions?

Reply:
if we remove assignee the count still increments

Reply:
so, does it meet the requirements?

Reply:
Yes, each change should increase counter by 1